docs/source/projects.rst

```diff
@@ -365,6 +370,84 @@ where ``<uri>`` is a Git repository URI or a folder. You can pass Git credential
 ``git-username`` and ``git-password`` arguments or using the ``MLFLOW_GIT_USERNAME`` and
 ``MLFLOW_GIT_PASSWORD`` environment variables.
 
+.. _kubernetes_execution:
+
+Run a project on Kubernetes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+MLflow projects can be executed in kubernetes clusters. Basically it uses the image created to run 
+projects in :ref:`Docker environment <project-docker-container-environments>`  and pushes it to an 
+image repository, so you need to configure MLproject with ``docker_env`` section. After that it 
+creates a Kubernetes Job that uses this published image and runs the MLflow project on kubernetes.
+A brief overview of how to configure and use this feature is as follows:
+
+In project folder you need to create a ``backend_config.json`` with the following attributes:
+
+.. code-block:: json
+
+{
+  "kube-context": "docker-for-desktop",
+  
+  "image-uri": "username/mlflow-kubernetes-example",
+
+  "kube-job-template-path": "kubernetes_job_template.yaml"
+}
+
+The ``kube-context`` attribute is the kubernetes context where mlflow will run the Job. ``image-uri`` points to the 
+registry/repository/image where the image will be pushed so kubernetes can download it and run. Remeber that mlflow 
+expects that login credentials are already stored for both kubernetes context and docker repository to push images.
+
+The ``kube-job-template-path`` points to a yaml file with the kubernetes Job/Batch specification to run the traning on 
+kubernetes. 
+See below the example available in the Docker example project. For more information about specification options please see 
+Kubernetes docs:
+`Jobs - Run to Completion <https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/>`_
+
+Have in mind that mlflow overwrites the folowing attributes in the job yaml file so it can handle jobs creation and monitor its status:
+
+- ``metadata.name``;
+
+- ``spec.template.spec.container.name``;
+
+- ``spec.template.spec.container.image``;
+
+- ``spec.template.spec.container.command``;
+
+
+.. code-block:: json
+
+  apiVersion: batch/v1
+  kind: Job
+  metadata:
+    name: ""
+    namespace: mlflow
+  spec:
+    ttlSecondsAfterFinished: 100
```

Comment:
we should include a `backoffLimit: 0` here by default.

Reply:
Done.